### PR TITLE
Checkers on docker_cli_pull_local_test.go

### PR DESCRIPTION
**Pull request by aditirajagopal**
Applying #16756 to integration-cli/docker_cli_pull_local_test.go

Signed-off-by: Aditi Rajagopal arajagopal@us.ibm.com

### CODE REVIEW

**Comment by aaronlehmann** [comment]
I don't understand the reason for this change. Can you please elaborate on why `checker.NotNil` should be used instead of `check.NotNil`?

**Comment by MHBauer** [comment]
`checker` the package has more available check methods. This change is to be consistent with all the other changes.

**Comment by calavera** [comment]
LGTM

File: integration-cli/docker_cli_pull_local_test.go [hunk 1]
```diff
@@ -149,13 +150,13 @@ func (s *DockerRegistrySuite) TestConcurrentPullMultipleTags(c *check.C) {
...
-		c.Assert(err, check.IsNil, check.Commentf("concurrent pull failed with error: %v", err))
+		c.Assert(err, checker.IsNil, check.Commentf("concurrent pull failed with error: %v", err))
 	}
 
 	// Ensure all tags were pulled successfully
 	for _, repo := range repos {
 		dockerCmd(c, "inspect", repo)
 		out, _ := dockerCmd(c, "run", "--rm", repo)
-		c.Assert(strings.TrimSpace(out), check.Equals, "/bin/sh -c echo "+repo, check.Commentf("CMD did not contain /bin/sh -c echo %s; %s", repo, out))
+		c.Assert(strings.TrimSpace(out), checker.Equals, "/bin/sh -c echo "+repo, check.Commentf("CMD did not contain /bin/sh -c echo %s; %s", repo, out))
```
Comment (runcom) [inline integration-cli/docker_cli_pull_local_test.go]: Commentf can be removed I guess

File: integration-cli/docker_cli_pull_local_test.go [hunk 2]
```diff
@@ -77,14 +78,14 @@ func (s *DockerRegistrySuite) TestConcurrentPullWholeRepo(c *check.C) {
...
-		c.Assert(err, check.IsNil, check.Commentf("concurrent pull failed with error: %v", err))
+		c.Assert(err, checker.IsNil, check.Commentf("concurrent pull failed with error: %v", err))
 	}
 
 	// Ensure all tags were pulled successfully
 	for _, repo := range repos {
 		dockerCmd(c, "inspect", repo)
 		out, _ := dockerCmd(c, "run", "--rm", repo)
-		c.Assert(strings.TrimSpace(out), check.Equals, "/bin/sh -c echo "+repo, check.Commentf("CMD did not contain /bin/sh -c echo %s: %s", repo, out))
+		c.Assert(strings.TrimSpace(out), checker.Equals, "/bin/sh -c echo "+repo, check.Commentf("CMD did not contain /bin/sh -c echo %s: %s", repo, out))
```
Comment (runcom) [inline integration-cli/docker_cli_pull_local_test.go]: here as well

**Comment by runcom** [comment]
I'll fix my nits in a follow up PR 

LGTM

**Pull request merged**